src/sentry/mail/notifications.py

```diff
@@ -137,13 +137,15 @@ def get_builder_args(
     # TODO: move context logic to single notification class method
     extra_context = (extra_context_by_actor_id or {}).get(recipient.actor_id, {})
     context = get_context(notification, recipient, shared_context or {}, extra_context)
+
+    html_template, txt_template = notification.get_email_template_filenames()
```

Comment:
@mgaeta I don't quite understand this change, what was wrong with having two different methods like before?